src/models/SqlOutputContentProvider.ts

```diff
@@ -78,8 +78,11 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
             } catch (e) {
                 prod = false;
             }
+            let mssqlConfig = self._vscodeWrapper.getConfiguration('mssql');
```

Comment:
It's cleaner to have a single line that gets the fontFamily object, then a 2nd line that does the split and join operations rather than duplicating these. Ideally we'd avoid double-reading from the mssqlConfig also